spark-extension-shims-spark/src/test/scala/org/apache/spark/sql/auron/BuildinfoInSparkUISuite.scala

```diff
@@ -17,12 +17,18 @@
 package org.apache.spark.sql.auron
 
 import org.apache.spark.sql.execution.ui.AuronSQLAppStatusListener
+import org.apache.spark.util.Utils
 
 class BuildinfoInSparkUISuite
     extends org.apache.spark.sql.QueryTest
     with BuildInfoAuronSQLSuite
     with AuronSQLTestHelper {
 
+  override protected def beforeAll(): Unit = {
+    super.beforeAll()
+    val eventLogDir = Utils.createTempDir("/tmp/spark-events")
```

Comment:
Taking a look again, I think the order here needs to be flipped. The directory is required for the spark context creation, otherwise the tests will fail. 
```suggestion
    val eventLogDir = Utils.createTempDir("/tmp/spark-events")
    super.beforeAll()
```